### PR TITLE
Task/T-14452 add relay meter helm chart

### PR DESCRIPTION
**Pull request by rutwickjain87**
Helm chart to deploy POKT Relay Meter service on EKS

### CODE REVIEW

File: charts/pokt-relay-meter/values.yaml [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# Default values for pokt-relay-meter
+# This is a YAML-formatted file
+# Declare variables to be passed into your templates
+
+replicaCount: 1
+
+collector:
+  image:
+    repository: 059424750518.dkr.ecr.us-west-2.amazonaws.com/relay-meter-collector
```
Comment (WesleyCharlesBlake) [inline charts/pokt-relay-meter/values.yaml:9]: use `pocketfoundation/relay-meter` when its pushed

File: charts/pokt-relay-meter/values.yaml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    pullPolicy: IfNotPresent
+    tag: "latest"
```
Comment (WesleyCharlesBlake) [inline charts/pokt-relay-meter/values.yaml:11]: I would leave this blank, and then set the appVersion in Charts.yaml to match the semver tag

**Review by WesleyCharlesBlake — CHANGES_REQUESTED** [review]
there a bit that needs changing here. I'll push some changes in a bit

File: charts/pokt-relay-meter/templates/tests/test-connection.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+kind: Pod
+metadata:
+  name: "{{ include "pokt-relay-meter.fullname" . }}-test-connection"
+  labels:
+    {{- include "pokt-relay-meter.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: wget
```
Comment (WesleyCharlesBlake) [inline charts/pokt-relay-meter/templates/tests/test-connection.yaml:11]: change the container name to `nc`

**Review by WesleyCharlesBlake — DISMISSED** [review]
LGTM, just change the container name for the test-connection pod to something more applicable

File: charts/pokt-relay-meter/templates/tests/test-connection.yaml (hunk 3 above)
Comment (rutwickjain87) [reply charts/pokt-relay-meter/templates/tests/test-connection.yaml]: fixed

**Review by WesleyCharlesBlake — APPROVED** [review]

**Pull request merged**